markdown/counselor-submission-faq.md

```diff
@@ -3,24 +3,26 @@ title: Session Submission FAQs
 description: Answers common questions around submitting a session to speak at THAT Conference.
 ---
 
-We've collected a number of questions that we feel may (or have already been) frequently asked surrounding the process of submitting a session to speak at THAT Conference. If your question isn't answered here feel free to reach out in Slack in the #sessions channel or via e-mail at [Hello@THATConference.com](mailto:Hello@THATConference).
+We've collected a number of questions that we feel may (or have already been) frequently asked surrounding the process of submitting a session to speak at THAT Conference. If your question isn't answered here feel free to reach out in Slack in the #sessions channel or via e-mail at [hello@THATConference.com](mailto:hello@THATConference).
 
 ### Why do I need a new account?
 
 After THAT Conference 2019, we deleted our old site and created a new one from the ground up. Everything went with it, including user accounts. You have to create a new account on the site.
 
 ### What happened to my old sessions?
 
-After THAT Conference 2019, we deleted our old site and created a new one from the ground up. Everything went with it, including sessions. We may decide to migrate the old data over to the new site and connect it with your profile. Things are still under development so we're not 100% sure what will happen.
+After THAT Conference 2019, we deleted our old site and created a new one from the ground up. Everything went with it, including sessions. We are currently working to build out the rest of the site functionality, including migrating over past sessions and speakers. [Sign up for our newsletter](#newsletter), [join THAT Slack](https://thatslack.thatconference.com/) and/or watch our social media as we announce new site functionality as we release it.
+
+We may decide to migrate the old data over to the new site and connect it with your profile. Things are still under development so we're not 100% sure what will happen.
```

Comment:
How about "Things are still under development so we're not 100% sure the exact timelines or details on the migration."